docs/getting-started/user-guides/dcrd-setup.md

```diff
@@ -23,13 +23,21 @@ or `tmux`.
 
 ## **<i class="fa fa-cloud"></i> Connect to the Decred Network**
 
-> Open dcrd Using the --rpcuser and --rpcpass Parameters
+The first time launching `dcrd`, it will connect to the Decred network via multiple peers and begin downloading the blockchain. To allow `dcrwallet` and `dcrctl` to communicate with `dcrd`, the configuration files must have `rpcuser` and `rpcpass` settings enabled. 
```

Comment:
Tehcnically we don't yet have multi-peer downloading yet so drop 'via multiple peers' maybe.